### PR TITLE
Thelio b4.0

### PR DESCRIPTION
**Pull request by jacobgkau**
This will add the Thelio b4.0 to tech docs.

### CODE REVIEW

**Comment by jacobgkau** [comment]
This is starting to feel like I could keep making revisions forever, but it's probably ready to review at this point.

I had to digitally alter the back overview to correct the inaccurate etched port labels (HDMI & DP version numbers), which I don't like doing. I don't know if we have an ETA on a corrected chassis being produced and available for a photo, though (and I believe the normal lights are gone from the photo room this week.) We can potentially swap that photo out later if what's there now doesn't look too bad.

One other item of note, I went with just `thelio-b4` for the model of this section as opposed to `thelio-b4.0` since the `.0`/`-0` formatting is a little inconsistent and we wouldn't typically do a new Tech Docs section for a minor revision anyway. If this leads to confusion, we can add `.0` or `-0` back in the future to clarify. I did add a note to the top of the repairs page about the revision similar to the ones we use for laptop pages regarding their screen size variations.

File: src/models/thelio-b4/external-overview.md
Comment (leviport) [inline src/models/thelio-b4/external-overview.md:5]: I believe the ethernet port should be labeled `2.5Gb/s` or `2.5Gb` rather than `2.5GiB`. GiB is gibibyte, and gibibyte != gigabyte != gigabit.

File: src/models/thelio-b4/repairs.md
Comment (leviport) [inline src/models/thelio-b4/repairs.md:75]: It looks like this image is missing from the img directory. I'm only seeing the description text.

File: src/models/thelio-b4/repairs.md
Comment (leviport) [inline src/models/thelio-b4/repairs.md:182]: CPU shroud link is empty

**Review by leviport — CHANGES_REQUESTED** [review]
I found a couple small bugs, but this is dynamite overall. I'll approve once those are taken care of

File: src/models/thelio-b4/external-overview.md
Comment (jacobgkau) [reply src/models/thelio-b4/external-overview.md:5]: You're right, I used the wrong abbreviation. [Microsoft's styling guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms) and the motherboard manual use `Gbps`, so I'll go with that. (Intel ARK uses `GbE`.)

Comment (jacobgkau) [reply src/models/thelio-b4/external-overview.md:5]: Fixed with https://github.com/system76/tech-docs/pull/153/commits/21c55ff87f2b786a5bf7bbd8cdda2f6d69b43dd0.

File: src/models/thelio-b4/repairs.md
Comment (jacobgkau) [reply src/models/thelio-b4/repairs.md]: Fixed with https://github.com/system76/tech-docs/pull/153/commits/6603dd6300d60022bf55253eb703098b484d37a4.

File: src/models/thelio-b4/repairs.md
Comment (jacobgkau) [reply src/models/thelio-b4/repairs.md:75]: I forgot that photo had a focus issue. I retook it today and will drop it in once I can get it off of my camera.

**Review by leviport — APPROVED** [review]
Looks great, nice work! I'll let you merge this once you're ready.

File: src/models/thelio-b4/repairs.md
Comment (ahoneybun) [inline src/models/thelio-b4/repairs.md:75]: I believe this should be: 25-drive-installed.webp

**Review by ahoneybun — CHANGES_REQUESTED** [review]
Some of the image filenames seem to be incorrect causing them to not load.

**Review by ahoneybun — APPROVED** [review]
This is just fantastic! I love the section for the Thelio Io board wiring!

**Pull request merged**
